Libraries/Hotcakes.Shipping.UpsFreight/BillingOption.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
```

Comment:
This is missing the open source license header

Reply:
done. added license header.